Development/nmos/mdns.cpp

```diff
@@ -106,13 +107,22 @@ namespace nmos
     }
 
     bool get_service_authorization(const nmos::service_type& service, const nmos::settings& settings)
+    {
+        // IS-09 System API does not use authorization
+        // See https://github.com/AMWA-TV/is-09/issues/21
+        // IS-10 Authorization API does not use authorization
```

Comment:
Is there a reference for the IS-10 Authorization API not using authorization??

Reply:
This is referencing the Authorization API which is the API on the Authorization Server. Let's remove the `IS-10` from the comment.